tool/jt.rb

```diff
@@ -517,6 +518,26 @@ def dis(file)
     puts Pathname(file).sub_ext('.ll')
   end
 
+  def environment
+    puts "Environment"
+    env_vars = ['JAVA_HOME','RUBY_BIN','GRAALVM_BIN','GRAAL_HOME',
+    'JVMCI_BIN','JVMCI_GRAAL_HOME','GRAAL_JS_JAR',
+    'SL_JAR','OPENSSL_PREFIX','AOT_BIN','TRUFFLERUBY_CEXT_ENABLED',
+    'TRUFFLERUBYOPT','RUBYOPT']
+    env_vars.each do |e|
+      puts "#{e}: #{ENV[e]}"
+    end
+    raw_sh('ruby -v', continue_on_failure: true)
```

Comment:
You have to split those arguments:
```ruby
raw_sh('ruby', '-v', continue_on_failure: true)
```

You could create a helper lambda in this method if want:
```ruby
shell = -> command { raw_sh(*command.split, continue_on_failure: true) }
shell['ruby -v'] # or .call()
```

Reply:
Fixed. Used the lambda. Thanks.